src/sql_types.rs

```diff
@@ -88,7 +88,7 @@ pub struct Function {
 }
 
 impl Function {
-    pub fn args(&self) -> impl Iterator<Item = (u32, &str, Option<&str>, Option<String>)> {
+    pub fn args(&self) -> impl Iterator<Item = (u32, &str, Option<&str>, Option<(String, bool)>)> {
```

Comment:
Could you pls create a struct for this so the values have associated names? It took me a while to figure out exactly what the bool represented
```
Option<(String, bool)
```

Reply:
Agree, tuple was hard to read. Replaced with an enum.